### ISSUE TITLE
share loss images

### ISSUE THREAD

**Issue by AngelGuevara7** [myshell-ai/MeloTTS#67]
Hi!
Could you share the loss images during training to get an idea of how they should look like? 
I'm trying to train a new single speaker model but my model cant articulate words at early stages (epoch 500) eventhough the attention matrix looks diagonal. 
I attach the config file in case it might help.
Thanks!!

[config.json](https://github.com/myshell-ai/MeloTTS/files/14590269/config.json)

**Comment by jeremy110** [comment]
How many hours did you train on?
In my case, it took about 5~8 hours  data to train, but I haven't tried fewer hours.
![image](https://github.com/myshell-ai/MeloTTS/assets/52516810/e798e715-233a-44b8-b7ff-c18136f5b124)

**Comment by AngelGuevara7** [comment]
Hey, thanks for sharing!
Nevermind, it was a problem with my data. I resampled my 22,5kHz data to 44,1kHz and there where some artifacts in the high frequencies, that was the problem. Changing the frequency to 22,5kHz solved the problem and now it is sounding great!
In case it helps to anybody, I trained with 20 hours data and below I let the generator losses, which are higher than yours but it still sounds great to me. 
![g_loss](https://github.com/myshell-ai/MeloTTS/assets/93993439/41b67656-23a7-4d56-abb5-b0d14944533a)

**Comment by gansidui** [comment]
> Hey, thanks for sharing! Nevermind, it was a problem with my data. I resampled my 22,5kHz data to 44,1kHz and there where some artifacts in the high frequencies, that was the problem. Changing the frequency to 22,5kHz solved the problem and now it is sounding great! In case it helps to anybody, I trained with 20 hours data and below I let the generator losses, which are higher than yours but it still sounds great to me. ![g_loss](https://private-user-images.githubusercontent.com/93993439/313625790-41b67656-23a7-4d56-abb5-b0d14944533a.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY5MjM3NTMsIm5iZiI6MTcyNjkyMzQ1MywicGF0aCI6Ii85Mzk5MzQzOS8zMTM2MjU3OTAtNDFiNjc2NTYtMjNhNy00ZDU2LWFiYjUtYjBkMTQ5NDQ1MzNhLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIxVDEyNTczM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4MGY2NmI0Y2QzN2YzOWI1ZTQyY2U1ZGMzNjc0MWQ2ZWNiMzQ3NjEyMWY3MGFlODBmM2Y0MzA5ZDUzOTY4NWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.w32OdJdmLZlx55NUdTjps2yAtdw_Jp6KLoNiY-GwOqo)

Hello, the pre-trained model is 44.1kHz, and the fine-tuned data is 22.5kHz. There must be at least one resampling. Wouldn't it be better to resample the fine-tuned data to 44.1kHz first?

**Comment by iv2985** [comment]
Where does one find the loss or find/generate the loss images?

**Comment by jeremy110** [comment]
@iv2985 
In your tensorboard.
